docs/users_guide/glasgow_exts.rst

```diff
@@ -3590,8 +3590,7 @@ automatically derived:
    ``Functor``, defined in ``GHC.Base``. See :ref:`deriving-functor`.
 
 -  With :ghc-flag:`-XDeriveDataTypeable`, you can derive instances of the class
-   ``Data``, defined in ``Data.Data``. See :ref:`deriving-typeable` for
-   deriving ``Typeable``.
+   ``Data``, defined in ``Data.Data``. See :ref:`deriving-data`.
```

Comment:
Since `Typeable` no longer has anything to do with `DeriveDataTypeable`, I'm changing this link to point to the new "Deriving `Data` instances" section instead.